go/mysql/binlog_event_json_test.go

```diff
@@ -150,7 +153,7 @@ func TestJSON(t *testing.T) {
 				}
 			} else {
 				if got := string(r); got != tcase.expected {
-					t.Errorf("unexpected output for %v: got [%v] expected [%v]", tcase.data, got, tcase.expected)
+					t.Errorf("unexpected output for %v: got \n[%v] \n expected \n[%v]", tcase.data, got, tcase.expected)
```

Comment:
Is adding the newlines here intentional?

Reply:
Yeah. I also do this often. It makes the outputs align, easier to compare.